docs/content/en/docs/installation/_index.en.md

```diff
@@ -129,8 +133,10 @@ See [how to add token to kubeconfig]({{% ref "../trouble-shooting/setup/kubernet
 ```sh
 $ export FISSION_NAMESPACE="fission"
 $ kubectl create namespace $FISSION_NAMESPACE
-$ helm install --namespace $FISSION_NAMESPACE --name-template fission \
-    https://github.com/fission/fission/releases/download/{{% release-version %}}/fission-all-{{% release-version %}}.tgz
+$ kubectl create -k "github.com/fission/fission/crds/v1?ref={{% release-version %}}"
+$ helm repo add fission-charts https://fission.github.io/fission-charts/
+$ helm repo update
+$ helm install --namespace $FISSION_NAMESPACE  fission fission-charts/fission-all
```

Comment:
Version should be mentioned here. 
```
--version {{%release-version%}}
```

BotKube example
```
 $ helm install --version v0.12.1 botkube --namespace botkube \
  infracloudio/botkube
```

Same for all below.

Also, please see if you can separate installation of helm repo at very start. So that it would be a common part for rest of sections.

Reply:
I think it's better to repeat in this case, as users might miss copying the other stanzas.